web/modules/custom/nidirect_webforms/nidirect_webforms.module

```diff
@@ -234,3 +234,16 @@ function nidirect_webforms_preprocess_webform_confirmation(&$variables) {
     $variables['title'] = $title;
   }
 }
+
+/**
+ * Implements hook_cron().
+ */
+function nidirect_webforms_cron() {
+  $seven_weeks_ago = \Drupal::service('datetime.time')->getRequestTime() - (7 * 7 * 24 * 60 * 60);
+  $connection = \Drupal::database();
+
+  // Delete records older than 7 weeks.
+  $connection->delete('prison_visit_booking_link_ids')
```

Comment:
Can call this directly without creating a connection variable 
\Drupal::database()->delete()